src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -201,6 +210,26 @@ public ResponseEntity<UUID> createModificationInGroup(@RequestBody ModificationI
         return ResponseEntity.ok().body(networkModificationService.createModificationInGroup(modificationsInfos));
     }
 
+    @PutMapping(value = "/network-modifications", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
@PutMapping(value = "/network-modifications/stash" ?
A PUT request on /network-modifications should be a simple update